aiida_common_workflows/workflows/relax/cp2k/generator.py

```diff
@@ -190,7 +200,7 @@ def _construct_builder(self, **kwargs) -> engine.ProcessBuilder:
             builder.cp2k.kpoints = kpoints
 
         # Removing description.
-        _ = parameters.pop('description')
+        del parameters['description']
```

Comment:
rather than using the complete protocol and then removing what does not belong in a cp2k input, maybe it would be a better idea to move all the cp2k input below a separate subkey, such that the yaml which currently reads as

```yaml
moderate:
  description: 'Protocol to relax a structure with normal precision at moderate computational cost.'
  kpoints_distance: 0.5
  basis_pseudo: dzvp-pbe-gth.yml
  GLOBAL:
    PRINT_LEVEL: MEDIUM              #default: MEDIUM. Important to have the correct parsing
  FORCE_EVAL:
    METHOD:         QUICKSTEP
    STRESS_TENSOR:  ANALYTICAL
...
```

becomes

```yaml
moderate:
  description: 'Protocol to relax a structure with normal precision at moderate computational cost.'
  kpoints_distance: 0.5
  basis_pseudo: dzvp-pbe-gth.yml
  input:
    GLOBAL:
      PRINT_LEVEL: MEDIUM              #default: MEDIUM. Important to have the correct parsing
    FORCE_EVAL:
      METHOD:         QUICKSTEP
      STRESS_TENSOR:  ANALYTICAL
...
```

what do you think?

Reply:
I like the idea 👍 , I will implement that

Reply:
awesome :-) thanks!